lib/livebook/runtime/definitions.ex

```diff
@@ -183,6 +183,21 @@ defmodule Livebook.Runtime.Definitions do
           packages: [kino_explorer]
         }
       ]
+    },
+    %{
+      kind: "Elixir.Kino.RemoteCell",
+      name: "Remote cell",
```

Comment:
Other options: "Remote node", "Remote execution", "Remote procedure call", "Distributed Elixir".

The last one may be the most clear one.

Reply:
I like "Remote execution".

Whatever we pick, we should mirror the name in kino :)